sdl2/VITABUILD

```diff
@@ -1,7 +1,7 @@
 pkgname=sdl2
 pkgver=2.0.23
-pkgrel=3
-gitrev=0a1641635d4a378957b077dd45ecfdb63527be26
+pkgrel=4
+gitrev=9670d2bb9ed19e3720c58378fe3eadab1ea0c40e
```

Comment:
how about change a version to `2.23.2` and following release of sdl team?

- `cb46e1b` is `2.23.2`
- `9670d2` could be `2.23.2-1`

if you think that we need `9670d2`, change version to `2.23.2-1`
and if not, we can use `prerelease-2.23.2` tag or `cb46e1b`

Reply:
We'll switch to new numbering scheme when 2.24 gets released

Reply:
they have already changed lib version to 2.23.x https://github.com/libsdl-org/SDL/commit/f1416ef2ba
and now it has a pretty clear meaning that this is a dev/unstable version.  :)
https://github.com/libsdl-org/SDL/issues/5217#issuecomment-1112475879

Reply:
I know. But officially new numbering scheme should be used only since 2.24

Reply:
huh? so, why did they make a tag to 2.23.2?

Reply:
Fine

Reply:
thanks. LGTM :+1: